<!-- 
  Launch:  
-->
<launch>
        <include file="$(find osm_server)/launch/osm_path_generator_servis.launch"/>

	<node pkg="osm_server" type="osm_path_example.py" name="osm_path" output="screen"/>
          <!--param name="start_lat" value="16.2609" /-->
          <!--param name="start_lng" value="52.2324" /-->
          <!--param name="end_lat" value="16.2300" /-->
          <!--param name="end_lng" value="52.2392" /-->

          <param name="start_lat" value="16.25526" />
          <param name="start_lng" value="52.23454" />
          <param name="end_lat" value="16.2598" />
          <param name="end_lng" value="52.2330" />

          <param name="lat1" value="52.2408" />
          <param name="lat2" value="52.2052" />
          <param name="lng1" value="16.2620" />
          <param name="lng2" value="16.2264" />

          <param name="aeroway_filter" value="1.0" />

</launch>
